### ISSUE TITLE
Issue with Dropbox/Rclone/Plex or OmniS???

### ISSUE THREAD

**Issue by DomMuc83** [kelinger/OmniStream#56]
Hey guys, i‘m facing the issue that plex does not seem to save the „analyzed“ status of files…
After an omni down/up or after a server reboot all files need a new analyzing to work fast and proper…

Setup is with Dropbox..

anyone else? Any ideas?

thx for the great work 😁👍

**Comment by kelinger** [comment]
Unless I'm mistaken, this should be going into your configs which are stored locally, not on Dropbox.  If you type `omni configs` you should then be in a directory with a subdirectory for nearly all containers part of OmniStream.  One of those is for Plex and should have all your databases, preferences, and metadata.  Is this the case with your setup?

**Comment by DomMuc83** [comment]
Yes its there - and before switching to dropbox, rclone updates and latest omnistream upgrades everything seems to be fine for a very long time 🤪
Thats the point why i‘m confused… if no one else facing this - i have to take a deeper look…
Also Plex has problems with finding „files added to dropbox“ - Scan library dont bring them up… (only a complete new creation of that Library) - so i think it could be an error on „all systems“ 🤪🙈

**Comment by DomMuc83** [comment]
I think there is an issue with the mountpoint in OmniStream.
If i do some metadata Updates in Plex etc. there sometimes all crashes down and Plex is going in an „idle“ mode…
Then, if i am looking at my Server it get stuck on the „df -h“ command and get „idle“ too…
I think the omnimount „crashes“ somewhere…
But omnilogs shows nothing in my mind… 🤷

**Comment by DomMuc83** [comment]
Ok… - so for now the issue is gone…

2 Things, but dont know if anything really effects it…

- using „plex public tag“ instead of „plexpass“
- Setting Up Kernel Update to 6.1


So i close it for now, maybe opening again if it will be back 🤪

Thx guys.

**Comment by DomMuc83** [comment]
So i still have an issue with omnimount and dropbox…

everything is „healthy“ and running fine (jellyfin and plex) for the first „initial scan“ than after adding new files to the media folders in dropbox (where all the files are stored) whether plex nor jellyfin find these files with a scan triggered through their frontend…
So i do an „omni up / omni down“ the servers are back - and now the scan is going to need more time - and also find the new files in the dropbox folder…
But After omni up / omni down plex still is going to load some data if i click on a movie detail page… this slows everything down… and plex even got that data… its there… so i dont know what plex is doing there…

Any idea??

**Comment by DomMuc83** [comment]
A new Update: 

If i am doing something on the Plex Database (search new media etc.) the Dropbox mount fails and Plex is going into an Database error…
This sometimes happens earlier, sometimes later during Plex scans…
Omni unmount does not work then, cause the Container cannot be „killed“…
Maybe Mount options for dropbox? Very strange…

**Comment by DomMuc83** [comment]
A fresh install on a fresh server works perfectly and flawless until a „omni down / omni up“ - it worked perfect until a restart… Plex loads everything like a charm… but after a restart, the mount drops frequently and plex do the things ive described above… 🙈🤷

**Comment by christeasdale** [comment]
Dropbox is supposed to support rclone polling for changes but it doesn't seem to work as well as Google Drive. I am having similar issues trying to use pCloud with OmniStream.  I think OmniStream needs an advanced option to allow more adjustments to the rclone mount.  I think this would allow other cloud services besides Google Drive to work better.
